modules/apps/foundation/frontend-theme/theme3/theme-3-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,221 @@
+$primary-color: rgb(56, 129, 104);
+
+@font-face {
+    font-family: WorkSans;
+    src: url(Work_Sans/WorkSans-Regular.ttf);
+}
+@font-face {
+    font-family: Lato;
+    src: url(Lato/Lato-Regular.ttf);
+}
+
+html {
+    height: 100%;
+    font-family: WorkSans, Arial, sans-serif;
+
+    h2, h4, p {
+        
+    a {
+        color: #ccc;
+        &:hover {
+            color: black;
+            text-decoration: underline;
+
+        }
+    }
+}
+
+    li {
+        color: #ccc;
+    
+    a {
+
+        &:hover {
+            color: black;
+        }
+    }
+}
+
+    #wrapper {
+        margin: 0;
+        padding: 0;
+        border: 0;
+       
+
+        ::selection {
+            background-color: $primary-color;
+            color: black;
+        }
+
+        ::-moz-selection {
+            background: $primary-color;
+            color: black;
+        }
+    
+        #{$all-text-inputs-active}, #{$all-text-inputs-focus} {
+
+            border: 1px solid black;
+
+        }
+
+        #{$all-buttons} {
+            color: white;
+            background-color: black;
+            border-color: black;
+
+            &:hover {
+                background-color: rgba(0,0,0,0.8);
+
+            }
+        }
+
+        .list-group-item {
+
+            &:hover {
+                color: white;
+                background-color: $primary-color;
+            }
+        }
+
+        .portlet-content.portlet-content-editable {
+            
+            &:hover {
+                color: white;
```

Comment:
You use ```rgba```, hex values, and colors throughout.  We should rely on hex values in most cases, but rgba whenever needed.